### DISCUSSION TITLE
Filtering Table View by Bound Column

### DISCUSSION THREAD

**Discussion by Skiziix** [khoait/DCE.PCF#40] [General] [answered]
This tool is awesome, and I've found a great use for it on a project I'm working on. However, it's important to be able to filter data based on the relationships made with Polylookup. Currently, there is no way to filter table data based on the bound column, maybe in the future this would be a great feature to add. Maybe Polylookup could save the associated data as a comma-separated list in the bound column? This would at least give the user basic filtering options. If there is already a workaround for this please let me know! Thanks for this awesome tool Khoait!

**Comment by khoait** [comment, marked as answer]
Hi @Skiziix, selected items can be outputted to the bound field using `output selected text` property. Alternatively, you can also output selected items to the bound field as a JSON array by setting `Output Selected Field` property to the bound field logical name.

Please more in wiki here [https://github.com/khoait/DCE.PCF/wiki/PolyLookup#output-selected-items-as-text](https://github.com/khoait/DCE.PCF/wiki/PolyLookup#output-selected-items-as-text)

Hope it helps!

**Reply by Skiziix** [reply]
Thanks @khoait for the response! I'm not quite sure how to contribute to the documentation, as I'm new to the GitHub community, but it may be helpful to make it clear in the documentation that the field has to be added to the form again in order for the selected item to be outputted as text.

**Reply by khoait** [reply]
Thanks for your suggestion! Only if you want to view the value, you can add the bound field again as a normal text field. Otherwise, you just need one bound field on the form hosting the pcf component and the output will be saved in the field correctly.
